ecommerce_integrations/public/build.json

```diff
@@ -0,0 +1,9 @@
+{
+    "js/shopify.bundle.js": [
+        "public/js/shopify/importer.js"
```

Comment:
note to self: need to make sure if this will work with new build system.

Reply:
<img width="997" alt="Screenshot 2022-01-12 at 7 55 43 PM" src="https://user-images.githubusercontent.com/9079960/149158560-1037f482-2ac5-40dc-a642-f679d65a52ea.png">


Doesn't work with new build system. I am guessing you use version-13 branch.

Reply:
Here's the PR that added new build system.

IMO we can also just move all JS to page.js instead of bundling it. It will reduce complexity a lot.

Reply:
This page would be used occasionally, so not much benefit from bundling the files.

Reply:
> This page would be used occasionally, so not much benefit from bundling the files.

Yes. Makes sense. The idea was to create a base class that can be inherited to import products, customers, and other stuff. But now I don't know if it's worth it.